hadml/metrics/compare_fn.py

```diff
@@ -263,3 +262,15 @@ def __call__(
         plt.close("all")
 
         return out_images
+
+
+class CompareMultiHadronEventGan():
```

Comment:
Although we did it like this before, there is no need for `CompareMultiHadronEventGan` to be a separate class. Its very unlikely we will be using more than one plotting function, so we don't need the modularity. This just makes the code needlessly complicated.

We can draw the plots using a method of `MultiHadronEventGANModule`